client/src/pages/Profile/Profile.scss

```diff
@@ -421,17 +421,125 @@
 .profile-leader-dashboard-links {
   color: var(--text-dynamic);
   display: flex;
+  flex-direction: row;
   justify-content: center;
   align-items: center;
-  flex-direction: row;
-  margin-bottom: 50px;
-  margin-top: 25px;
   text-align: center;
-  padding: 0px 20px;
-  flex-wrap: wrap;
+  margin-left: 35px;
+  margin-right: 20px;
   @include devices(tablet) {
-    flex-wrap: unset;
     flex-direction: column;
+    margin: auto;
+  }
+  .profile-leader-dashboard-permissions-links {
+    width: 300px;
+    margin-right: auto;
+    @include devices(tablet) {
+      margin: auto;
+      width: 350px;
+    }
+  }
+  .profile-leader-dashboard-other-links {
+    display: flex;
+    flex-direction: row;
+    margin-left: auto;
+    width: 100%;
+    margin-bottom: 50px;
+    margin-bottom: 30px;
+    @include devices(tablet) {
+      margin: auto;
+      width: 100%;
+      margin-bottom: 100px;
+      margin-top: 15px;
+    }
+    .profile-leader-dashboard-data-links {
+      width: 100%;
+      .profile-leader-dashboard-data-title {
+        display: flex;
+        flex-direction: row;
+        border-radius: 6px;
+        border: 2px solid var(--purple-shades-muted);
+        cursor: pointer;
+        margin: auto;
+        .profile-leader-dashboard-data-icon {
+          width: 30px;
+          height: 30px;
+          fill: var(--yellow);
+          margin: auto 0 auto auto;
+          padding-bottom: 2px;
+          padding-right: 5px;
+        }
+        .profile-leader-dashboard-data-text {
+          color: white;
```

Comment:
instead of white, use `var(--text-dynamic)` so that it's still legible in light mode,
![image](https://github.com/UofT-Frosh-Orientation/orientation-website/assets/86009011/2ce0b2ad-e5ee-4ba6-a046-eae34dcfb8f9)

the yellow icon colour is also a little illegible, but it should be fine for now haha